.github/workflows/build.yaml

```diff
@@ -29,6 +29,12 @@ jobs:
       - run: make get-deps
         if: ${{ matrix.go >= '1.17' }}
```

Comment:
nit comment, non-bloking: Do we need to support >=1.17 or can we change this to 1.19 as well?